samples: Bluetooth: df: Add info about max ant sw pattern len Kconfig (#6952)
nrfconnect/sdk-nrf: samples/bluetooth/direction_finding_connectionless_rx/README.rst

If the number of switch-sample periods is greater than the number of stored switching patterns, then the radio loops back to the first pattern.

> The length of an antenna switch pattern is limited by :kconfig:`CONFIG_BT_CTLR_DF_MAX_ANT_SW_PATTERN_LEN`.

Reviewer: ```suggestion
The length of an antenna switch pattern is limited by the :kconfig:`CONFIG_BT_CTLR_DF_MAX_ANT_SW_PATTERN_LEN`.
```

Author: Done

> If the required length of the antenna switching pattern is greater than the default value of :kconfig:`CONFIG_BT_CTLR_DF_MAX_ANT_SW_PATTERN_LEN` add the Kconfig to a board configuration file.

Reviewer: ```suggestion
If the required length of the antenna switching pattern is greater than the default value of the :kconfig:`CONFIG_BT_CTLR_DF_MAX_ANT_SW_PATTERN_LEN`, set this option to required value in a board configuration file.
```

Reviewer: "The Kconfig" in this context is a little confusing to me

Author: I've changed it to:
```
If the required length of the antenna switching pattern is greater than the default value of :kconfig:`CONFIG_BT_CTLR_DF_MAX_ANT_SW_PATTERN_LEN`, set the :kconfig:`CONFIG_BT_CTLR_DF_MAX_ANT_SW_PATTERN_LEN` with required value in a board configuration file.
```

> The length of an antenna switch pattern is limited by the :kconfig:`CONFIG_BT_CTLR_DF_MAX_ANT_SW_PATTERN_LEN`.
> If the required length of the antenna switching pattern is greater than the default value of :kconfig:`CONFIG_BT_CTLR_DF_MAX_ANT_SW_PATTERN_LEN`, set the :kconfig:`CONFIG_BT_CTLR_DF_MAX_ANT_SW_PATTERN_LEN` with required value in a board configuration file.
> For example for :ref:`nRF52833 DK <ug_nrf52>` add :kconfig:`CONFIG_BT_CTLR_DF_MAX_ANT_SW_PATTERN_LEN`=N, where N is the required antenna switch pattern length, to :file:`nrf52833dk_nrf52833.conf`.
> 

Reviewer 2: ```suggestion
The length of the antenna switching pattern is limited by the :kconfig:`CONFIG_BT_CTLR_DF_MAX_ANT_SW_PATTERN_LEN` option.
If the required length of the antenna switching pattern is greater than the default value of :kconfig:`CONFIG_BT_CTLR_DF_MAX_ANT_SW_PATTERN_LEN`, set the :kconfig:`CONFIG_BT_CTLR_DF_MAX_ANT_SW_PATTERN_LEN` option to the required value in the board configuration file.
For example, for the :ref:`nRF52833 DK <ug_nrf52>` add :kconfig:`CONFIG_BT_CTLR_DF_MAX_ANT_SW_PATTERN_LEN`=N, where N is the required antenna switching pattern length, to the :file:`nrf52833dk_nrf52833.conf` file.

```
